cmd/create/machinepool/helper.go

```diff
@@ -73,7 +73,8 @@ func getSubnetOptions(r *rosa.Runtime, cluster *cmv1.Cluster) ([]string, error)
 	// Format subnet options
 	var subnetOptions []string
 	for _, subnet := range privateSubnets {
-		subnetOptions = append(subnetOptions, aws.SetSubnetOption(subnet))
+		s := subnet
+		subnetOptions = append(subnetOptions, aws.SetSubnetOption(&s))
```

Comment:
Can we please change the functions to use subnets instead of pointers? 
As AWS SDK previously returned a slice of pointers to subnets and now returns a slice of subnets, they changed the output of `DescribeSubnets`.